continuous_load_testing/proto/grpc_gcp/testing/messages.proto

```diff
@@ -0,0 +1,330 @@
+// Message definitions to be used by integration test service definitions.
+
+syntax = "proto3";
+
+package grpc_gcp.testing;
+
+option go_package = "continuous_load_testing/proto/grpc_gcp/testing";
+
+message BoolValue {
```

Comment:
Let's simplify the message proto by:
1. remove all types we don't use in our test.
2. remove all unnecessary fields.